Documentation/Books/Manual/Views/ArangoSearch/GettingStarted.md

```diff
@@ -0,0 +1,107 @@
+# Getting started with ArangoSearch views
+
+## The DDL configuration
+
+All DDL operations on Views can be done via JavaScript or REST calls. The DDL syntax follows the well established ArangoDB guidelines and thus is very similar between JavaScript and REST. This article uses the JavaScript syntax.
+
+Assume the following collections were intially defined in a database using the following commands:
+
+```javascript
```

Comment:
Does syntax highlighting work with `javascript` (is it an alias)? We use `js` elsewhere.